pygfx/utils/gltf.py

```diff
@@ -28,6 +28,7 @@ def load_gltf(path):
 
 def _parse_mesh(mesh):
     visual = mesh.visual
+    # todo: why?
     visual.uv = visual.uv * np.array([1, -1]) + np.array([0, 1])  # uv.y = 1 - uv.y
```

Comment:
@panxinmiao any idea about the reason for this texcoord flip?

Reply:
This is because the uv coordinate systems of wgpu and opengl are different. The wgpu uses the D3D and Metal coordinate systems. The coordinate origin is in the upper left corner, while the opengl coordinate origin is in the lower left corner. trimesh loads textures according to the opengl coordinate system.

Reply:
Let's put this in a comment.